### PR TITLE
Use new model classes

### PR DESCRIPTION
**Pull request by sfbayhacker**
These set of commits include changes to use new model classes in the service layer instead of converting from old model to new model in the rest layer.

### CODE REVIEW

File: src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
    * @return the concept code
    */
   public static String getConceptCode(List<Property> properties) {
-    List<String> results = getProperty("NHC0", properties);
+    List<String> results = getProperty("code", properties);
```
Comment (sfbayhacker) [inline src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java:44]: @bcarlsenca please check

File: src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ public static String getConceptCode(List<Property> properties) {
    * @return the preferred name
    */
   public static String getPreferredName(List<Property> properties) {
-    List<String> results = getProperty("P108", properties);
+    List<String> results = getProperty("Preferred_Name", properties);
```
Comment (sfbayhacker) [inline src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java:59]: @bcarlsenca please check

File: src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java (hunk 1 above)
Comment (bcarlsenca) [reply src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java:44]: looks good.

File: src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java (hunk 2 above)
Comment (bcarlsenca) [reply src/main/java/gov/nih/nci/evs/api/util/EVSUtils.java:59]: looks good.

**Pull request merged**